scripts/import_fleurymichon.pl

```diff
@@ -877,70 +877,10 @@
 	}
 	#exit;
 
-	# Process the fields
-
-	# Food category rules for sweeetened/sugared beverages
-	# French PNNS groups from categories
-
-	if ($server_domain =~ /openfoodfacts/) {
-		ProductOpener::Food::special_process_product($product_ref);
-	}
-
-	if (    (defined $product_ref->{nutriments}{"carbon-footprint"})
-		and ($product_ref->{nutriments}{"carbon-footprint"} ne ''))
-	{
-		push @{$product_ref->{"labels_hierarchy"}}, "en:carbon-footprint";
-		push @{$product_ref->{"labels_tags"}}, "en:carbon-footprint";
-	}
-
-	if ((defined $product_ref->{nutriments}{"glycemic-index"}) and ($product_ref->{nutriments}{"glycemic-index"} ne ''))
-	{
-		push @{$product_ref->{"labels_hierarchy"}}, "en:glycemic-index";
-		push @{$product_ref->{"labels_tags"}}, "en:glycemic-index";
-	}
-
-	# Language and language code / subsite
-
-	if (defined $product_ref->{lang}) {
-		$product_ref->{lc} = $product_ref->{lang};
-	}
-
-	if (not defined $lang_lc{$product_ref->{lc}}) {
-		$product_ref->{lc} = 'xx';
-	}
-
-	# For fields that can have different values in different languages, copy the main language value to the non suffixed field
-
-	foreach my $field (keys %language_fields) {
-		if ($field !~ /_image/) {
-			if (defined $product_ref->{$field . "_$product_ref->{lc}"}) {
-				$product_ref->{$field} = $product_ref->{$field . "_$product_ref->{lc}"};
-			}
-		}
-	}
-
-	# Ingredients classes
-	extract_ingredients_from_text($product_ref);
-	extract_ingredients_classes_from_text($product_ref);
-
-	compute_languages($product_ref);    # need languages for allergens detection
-	detect_allergens_from_text($product_ref);
+	$User_id = $editor_user_id;
```

Comment:
because it called compute_nutrition_score($product_ref); that I changed, it was part of a long list of things that I had copy pasted some years ago, and now we have analyze_and_enrich_product_data($product_ref, $response_ref); that does all of it.

Reply:
it's not related to the new Fleury Michon data we received